### PR TITLE
Adobe wasmcloud kubernetes better together

### PR DESCRIPTION
**Pull request by brooksmtownsend**
Added our collaboration blog with Adobe on Wasm and Kubernetes

### CODE REVIEW

File: content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+title: "Better Together: Building Efficient Microservices in Kubernetes using WebAssembly"
+image: "images/blogs/adobe-kubernetes/header.png"
+date: 2022-11-17T11:00:00-05:00
+author: "Sean Isom and Colin Murphy, Adobe"
+description: "Bringing two major CNCF projects together – WebAssembly and Kubernetes – promises greater agility and major efficiencies"
```
Comment (autodidaddict) [inline content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md]: this implies that webAssembly is a CNCF project... but the CNCF project here is wasmCloud

Comment (autodidaddict) [reply content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md]: maybe `Bringing two major CNCF projects together - wasmCloud and Kubernetes - ...`

File: content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+Having multi-tenant clusters has loads of operational benefits (flexibility, security), but efficiently managing resources for arbitrary workloads with different scaling and provisioning characteristics is challenging. A common assumption with containers is that they automatically enable one to run efficiently in the cloud. However, containers still have to map to a fixed size virtual machine - fine if we’re dealing with small, agile distributed systems that can perfectly bin-pack. But, when mixed with larger, more monolithic systems, overall efficiency can drop.
+
+What if there was a more lightweight model that could be almost instantly scaled as traffic scales up, giving more scheduling flexibility than a coarse-grained container? I’m reminded of the quote from Docker creator, Solomon Hykes:
+
+{{< aside >}}
```
Comment (autodidaddict) [inline content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md:19]: blog post bingo  ✅ , Solomon Hykes quote.

File: content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+[The applier is hosted here](https://github.com/cosmonic/kubernetes-applier/tree/main/service-applier), although you can use `wash` commands and don’t necessarily need to clone the repository just to use the applier.
+
+Let’s set up the example `echo` actor to get things running. We’ll need to create several providers and actors, then wire them up with link definitions. First start the actors and providers [with the current latest versions](https://github.com/wasmCloud/capability-providers#first-party-capability-providers):
+
+```shell
+wash ctl start provider wasmcloud.azurecr.io/applier:0.3.0
+wash ctl start provider wasmcloud.azurecr.io/nats_messaging:0.14.5
+wash ctl start provider wasmcloud.azurecr.io/httpserver:0.16.3
+wash ctl start actor wasmcloud.azurecr.io/service_applier:0.3.0
+wash ctl start actor wasmcloud.azurecr.io/echo:0.3.4
```
Comment (autodidaddict) [inline content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md:103]: Are these the right versions? `echo` is on 0.3.5

**Review by autodidaddict — APPROVED** [review]
LGTM, just one super tiny nit about the title.

File: content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md (hunk 3 above)
Comment (brooksmtownsend) [reply content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md:103]: Yeah these are fine

File: content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md (hunk 1 above)
Comment (brooksmtownsend) [reply content/english/blog/better-together-building-efficient-microservices-in-kubernetes-with-webassembly.md]: Fixed 👍🏻

**Pull request merged**